index.js

```diff
@@ -18,6 +18,8 @@ const cli = require('./bin/cli');
   @param {boolean} [options.namespace.create] - flag to create the namespace if it does not exist. Only applicable for the build and deploy command. Must be used with namespace.name
   @param {string} [options.namespace.name] - flag to specify the project namespace name to build/deploy into.  Overwrites any namespace settings in your OpenShift or Kubernetes configuration files
   @param {string} [options.imageTag] - set the version to use for the nodeshift/centos7-s2i-image.  Versions are docker hub tags: https://hub.docker.com/r/nodeshift/centos7-s2i-nodejs/tags/
+  @param {string} [options.outputImageStream] - the name of the Image Stream to output to.  Defualts to project name from package.json
```

Comment:
Minor typo `Defualts` maybe copy/past from previous

Reply:
yea,  probably same for all the copy/pastes in that file too :)